src/osp/Active/SysNewton.h

```diff
@@ -193,22 +192,34 @@ class SysNewton : public IDynamicSystem
     static void create_body(ActiveScene& rScene, ActiveEnt entity,
                             NewtonWorld const* nwtWorld);
 
+    /**
+     * Update the inertia properties of a rigid body
+     *
+     * Given an existing rigid body, computes and updates the mass matrix and
+     * center of mass. Entirely self contained, calls the other inertia
+     * functions in this class.
+     * @param entity [in] The rigid body to update
+     */
+    static void update_rigidbody_inertia(ActiveScene& rScene, ActiveEnt entity);
```

Comment:
This looks like an update function when it isn't. Should this be "compute_" like the other functions?

Reply:
Yep